syft/pkg/cataloger/homebrew/package.go

```diff
@@ -0,0 +1,41 @@
+package homebrew
+
+import (
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+func newHomebrewPackage(name, version, desc, homepage string, locations file.LocationSet) pkg.Package {
+	p := pkg.Package{
+		Name:      name,
+		Version:   version,
+		Type:      pkg.HomebrewPkg,
+		Locations: locations,
+		Language:  pkg.Ruby,
+		FoundBy:   "homebrew-cataloger",
+		PURL:      packageURL(name, version),
+		Metadata: pkg.HomebrewMetadata{
+			Name:        name,
+			FullName:    name,
+			Tap:         "homebrew/core",
+			Homepage:    homepage,
+			Description: desc,
+		},
+	}
+
+	p.SetID()
+	return p
+}
+
+func packageURL(name, version string) string {
+	purl := packageurl.NewPackageURL(
+		"homebrew",
```

Comment:
It looks like there is a reference to `brew`, as a potential PURL type: https://github.com/package-url/purl-spec/blob/main/PURL-TYPES.rst#other-candidate-types-to-define, maybe we should use this instead?